src/bin/bat/clap_app.rs

```diff
@@ -257,6 +257,15 @@ pub fn build_app(interactive_output: bool) -> ClapApp<'static, 'static> {
                      an interactive terminal is detected. Possible values: *auto*, never, always.",
                 ),
         )
+        .arg(
+            Arg::with_name("always-decorations")
+                .short("f")
+                .alias("always-decor")
+                .overrides_with("always-decorations")
+                .hidden(true)
```

Comment:
Hiding from short `-h` text is fine, but I think this should appear in the long `--help` text.